en/flight_modes_mc/position_slow.md

```diff
@@ -0,0 +1,91 @@
+# Position Slow Mode (Multicopter) <Badge type="warning" text="main (v1.15+)" vertical="top" />
```

Comment:
FYI only. Trying something new here. A version introduced in "badge" to show when things appear. On release I'll upstate to actually say "v1.15"

![image](https://github.com/PX4/PX4-user_guide/assets/5368500/c98a7ceb-0004-4020-a199-4e1d5956e10f)

Generally this isn't all that necessary, but it can be. I'll be adding to actuators etc for v1.14.